test/js/inline-style-property-map.js

```diff
@@ -46,4 +46,32 @@ suite('Inline StylePropertyMap', function() {
 
     assert.throw(function() {inlineStyleMap.set('lemons', new SimpleLength(3, 'px'))}, TypeError);
   });
+
+  test('The delete method removes any StyleValue assigned to the CSS style property inputed', function() {
+    var inlineStyleMap = this.element.styleMap();
+    this.element.style.height = '10px';
+    inlineStyleMap.delete('height');
+
+    assert.strictEqual(this.element.style['height'], '');
+  });
+
+  test('The delete method should throw a TypeError if an unsupported property is given as input', function() {
```

Comment:
For brevity, you can just write 

'The delete method should throw a TypeError if an unsupported property'